pkg/kv/dynamodb/store.go

```diff
@@ -530,3 +528,12 @@ func (s *Store) StopPeriodicCheck() {
 		s.cancel = nil
 	}
 }
+
+func handleClientError(err error) error {
```

Comment:
This doesn't handle anything really.  Perhaps
```suggestion
func maybeExtractAWSErr(err error) error {
```
would be a better name?

Reply:
Makes sense 😄.  Unfortunately 😢 .